### ISSUE TITLE
Wrap task names in graph view

### ISSUE THREAD

**Issue by jfrost-mo** [cylc/cylc-ui#1942]
### Problem

Long task names are common in cylc, especially where tasks are generated through jinja and whatnot. In the graph view these long task names are displayed on a single line, which makes them very wide, and causes the workflow to spread out such that it is hard to follow. To read the text easily you need to zoom in, but then you can't see much of the overall workflow.

### Proposed Solution

The task names should be wrapped after a certain maximum length.

It should possibly also be truncated beyond a few lines to prevent ridiculously long names from breaking the UI layout, but this limit should be quite long so useful task name information is not lost.

**Comment by oliver-sanders** [comment]
Wrapping the task names would reduce the horizontal space required, however, it would also make them harder to read (due to the arbitrary break), which would likely result in complaints.

I think multiple line breaks would probably mess the layout a bit.

Workarounds:

* Transpose the graph. This reduces the impact of longer task names. The ability to set this as a global default will arrive one day.

Alternative solutions:

* Consider an option to use the task's `[meta]title` rather than full task name.
* Consider an option for task name reduction similar to what some text editors use for tabs.

**Comment by hjoliver** [comment]
I agree that the status quo is annoying if task names are long. Well, it's really the long names that are the problem, but it's understandable that people do that. 

👍  for the "alternative" solutions.
